atlas/telemetry/langchain_callback.py

```
@@ -0,0 +1,180 @@
+
+_HANDLER: TelemetryCallbackHandler | None = None
+_HANDLER_VAR: ContextVar[TelemetryCallbackHandler | None] = ContextVar(
+    "atlas_telemetry_callback",
+    default=None,
+)
+
+
+def configure_langchain_callbacks() -> None:
+    """Register the telemetry callback with LangChain once per process."""
+
+    global _HANDLER  # noqa: PLW0603
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `PLW0603`)

Remove unused `noqa` directive

(RUF100)